server/api/src/rsRmColl.cpp

```diff
@@ -746,14 +745,13 @@ rsMvCollToTrash( rsComm_t *rsComm, collInp_t *rmCollInp ) {
         }
     }
 
-    memset( &dataObjRenameInp, 0, sizeof( dataObjRenameInp ) );
+    dataObjCopyInp_t dataObjRenameInp{};
+    irods::at_scope_exit clear_dataObjCopyInp{[&dataObjRenameInp] { clearDataObjCopyInp(&dataObjRenameInp); }};
 
-    dataObjRenameInp.srcDataObjInp.oprType =
-        dataObjRenameInp.destDataObjInp.oprType = RENAME_COLL;
+    dataObjRenameInp.srcDataObjInp.oprType = dataObjRenameInp.destDataObjInp.oprType = RENAME_COLL;
```

Comment:
on two lines instead of one?

optimize readability...

Reply:
Done